packages/common/package.json

```diff
@@ -13,21 +17,17 @@
     "prettier": "prettier --check src/ tests/",
     "prettier:fix": "prettier --write src/ tests/",
     "lint": "eslint . --ext .ts,.tsx",
-    "lint:fix": "yarn lint --fix",
+    "lint:fix": "pnpm lint --fix",
     "verify": "run-p prettier lint",
-    "verify:fix": "yarn prettier:fix && yarn lint:fix",
+    "verify:fix": "pnpm prettier:fix && pnpm lint:fix",
     "build": "node ./build.js",
-    "postbuild": "tsc --build tsconfig.build.json",
-    "update-main-to-dist": "node ./update-pkg.js --update-main-to-dist",
-    "update-main-to-src": "node ./update-pkg.js --update-main-to-src",
-    "prepublish": "npm run update-main-to-dist",
-    "postpublish": "npm run update-main-to-src"
+    "postbuild": "tsc --build tsconfig.build.json"
   },
   "dependencies": {
-    "@atproto/common-web": "*",
+    "@atproto/common-web": "workspace:^",
     "@ipld/dag-cbor": "^7.0.3",
     "cbor-x": "^1.5.1",
-    "multiformats": "^9.6.4",
-    "pino": "^8.6.1"
+    "multiformats": "^9.9.0",
+    "pino": "^8.15.0"
```

Comment:
Was seeing issues with mismatched versions of Pino too, so bumped this everywhere too. Based on their changelog I don't think we'll see issues.